objectRecognitionSource/src/main/java/edu/ml/tensorflow/ObjectDetector.java

```diff
@@ -249,19 +248,21 @@ private Graph createNormalizerGraph() {
      * ADDED BY NAMIR - Used to convert recognitions to JSON
      * @param recognitions
      */
-    private List<Map<String, ?>> returnJSON(final List<Recognition> recognitions) {
+    private List<Map<String, ?>> returnJSON(final List<Recognition> recognitions, BufferedImage buffImage) {
         List<Map<String, ?>> jsonRecognitions = new ArrayList<>();
         for (Recognition recognition : recognitions) {
-            //recognition.getTitle(), recognition.getConfidence(), recognition.getLocation());
         	HashMap map = new HashMap();
         	map.put("label", recognition.getTitle());
         	map.put("confidence", recognition.getConfidence());
         	
+        	float scaleX = (float) buffImage.getWidth() / (float) SIZE;
+            float scaleY = (float) buffImage.getHeight() / (float) SIZE;
+        	
         	HashMap location = new HashMap();
-        	location.put("left", recognition.getLocation().getLeft());
-        	location.put("top", recognition.getLocation().getTop());
-        	location.put("right", recognition.getLocation().getRight());
-        	location.put("bottom", recognition.getLocation().getBottom());
+        	location.put("left", recognition.getScaledLocation(scaleX, scaleY).getLeft());
+            location.put("top", recognition.getScaledLocation(scaleX, scaleY).getTop());
+            location.put("right", recognition.getScaledLocation(scaleX, scaleY).getRight());
+            location.put("bottom", recognition.getScaledLocation(scaleX, scaleY).getBottom());
         	map.put("location", location);
```

Comment:
From this, it looks like we were returning the unscaled location?  That, returning it directly from the recognition rather than scaling to the image we're saving (either locally or VANTIQ-ly)?

Reply:
Yes exactly, we were in fact returning the unscaled location.